README.md

```diff
@@ -28,6 +28,11 @@ See the official [React Native website](https://facebook.github.io/react-native/
 ## Getting Started
 
 - Follow the [Getting Started guide](http://facebook.github.io/react-native/docs/getting-started.html) to install React Native and its dependencies.
+- Install the [Chocolatey package manager](https://chocolatey.org/) in an Administrator command prompt or PowerShell
+  * In the same Administrator command prompt, type ``choco install nodejs.install``
```

Comment:
I wonder if there's a more logical way to organize this.  We now have choco install instructions spanning to sections.  Maybe it would be better to just have an explicit section that says **Requirements**, with all tool dependencies and their version numbers listed, and a sub-section **Setting up the environment with Chocolatey** to signal that using Chocolatey is completely optional.